### ISSUE TITLE
Scary code block autocorrect each time

### ISSUE THREAD

**Issue by tas50** [rubocop-hq/rubocop#8179]
##  Expected behavior

Code should either autocorrect or not.

## Actual behavior

Code is marked as corrected, but not actually changed.

## Steps to reproduce the problem

This scary block autocorrect each run of RuboCop.

```ruby
agent = ossec_agents.find do |n| (

          
          n[:ossec][:agents].key?(agent_id) \
                          && n[:ossec][:agent][:enable])
end
```

```
/Users/tsmith/Desktop/test.rb:3:1: W: Lint/UselessAssignment: Useless assignment to variable - agent.
agent = ossec_agents.find do |n| (
^^^^^
/Users/tsmith/Desktop/test.rb:3:34: C: [Corrected] Layout/MultilineBlockLayout: Block body expression is on the same line as the block start.
agent = ossec_agents.find do |n| ( ...
                                 ^
/Users/tsmith/Desktop/test.rb:4:1: C: [Corrected] Layout/EmptyLinesAroundBlockBody: Extra empty line detected at block body beginning.
/Users/tsmith/Desktop/test.rb:5:1: C: [Corrected] Layout/TrailingWhitespace: Trailing whitespace detected.
```

## RuboCop version

Master
